services/api-server/src/simcore_service_api_server/models/schemas/studies.py

```diff
@@ -0,0 +1,15 @@
+from models_library.projects import ProjectID
+from models_library.projects_nodes_io import NodeID
+from pydantic import Field
+
+from .solvers import SolverPort
+
+StudyID = ProjectID
+
+
+class StudyPort(SolverPort):
+    key: NodeID = Field(
```

Comment:
Q: the key of a StudyPort is a NodeID ??? is this correct?

Reply:
This is correct. I want to make sure that the key has the same format as the NodeID since the SolverPort ``key`` has other constraints. I can make an alias ``PortID`` for clarification